pkg/prowgen/prowgen.go

```diff
@@ -193,14 +193,18 @@ func generatePresubmitForTest(jobBaseBuilder *prowJobBaseBuilder, name string, i
 
 	shortName := info.TestName(name)
 	base := jobBaseBuilder.Rehearsable(!opts.disableRehearsal).Build(jc.PresubmitPrefix)
+	rerunCommand := prowconfig.DefaultRerunCommandFor(shortName)
+	if opts.defaultDisable {
```

Comment:
don't we also want to verify that the job isn't `optional` here?

Reply:
conditions updated